src/security.cs

```diff
@@ -183,7 +183,7 @@ enum SecKeyClass {
 	}
 
 	[Static][Internal]
-	interface KeysAuthenticationType {
+	interface SecAuthenticationTypeKey {
```

Comment:
The name refactoring seems legit to me 👍 

_Note: we usually try not to rename things the avoid noise and mess the git history but we can in current master because of some recent (super noisy) PMCS changes._

Reply:
Agree, I like the changes